README.md

```diff
@@ -4,17 +4,29 @@ Official Prettier plugin adding support for formatting `.astro` files.
 
 ## Installation
 
+First install Prettier and the plugin:
+
 ```shell
-npm i --save-dev prettier-plugin-astro prettier
+npm i --save-dev prettier prettier-plugin-astro
+```
+
+Then add the plugin to your Prettier configuration:
```

Comment:
why not combine this with the following paragraph?

Reply:
Since needing to put plugins explicitly in your config is a Prettier 3 breakage and a lot of people seems to have missed it, I kinda want to make sure it's clear for users that they *need* to do it, it's not just recommended (unlike the following config, which is optional, but recommended if you encounter issues (aka, you're using pnpm))